packages/l2-block-verification-in-assertion/README.md

```diff
@@ -0,0 +1,29 @@
+# L2 block verification in assertion
+
+This tutorial shows how to verify whether an L2 block has been processed as part of an RBlock assertion on L1.
+
+It uses the `Rollup` contract to find the latest confirmed (or created if configured in the script) RBlock/node, find the event that created that RBlock/node and get the latest processed L2 block hash as part of that assertion.
```

Comment:
find the event that created that RBlock/node and get the latest processed L2 block hash as part of that assertion.

Should we replace assertion with RBlock/node to avoid confusions?

Reply:
The assertion is one part of the RBlock/node, but I'll change the wording to make it clearer. Thanks! 🙏

Reply:
Thank you!